### ISSUE TITLE
Non-workbench editor does not show highlighting

### ISSUE THREAD

**Issue by atkinchris** [AdamCaviness/vs-code-theme-monokai-dark-soda#4]
The highlight colour in Monokai Dark Soda is the same colour as the background of non-workbench panels, so highlights cannot be seen. This appears to have happened when VS Code switched to using the Monaco editor for settings search, source control and more.

Default theme:
![Screenshot 2020-03-02 at 14 48 09](https://user-images.githubusercontent.com/1479528/75687096-2fcbf680-5c95-11ea-8825-a807930efd97.png)

Monokai Dark Soda:
![Screenshot 2020-03-02 at 14 47 56](https://user-images.githubusercontent.com/1479528/75687125-3ce8e580-5c95-11ea-9fb6-676fb23e1d5d.png)

For reference, this is the highlighting colour when used in a workbench window:
![Screenshot 2020-03-02 at 14 51 35](https://user-images.githubusercontent.com/1479528/75687223-5db13b00-5c95-11ea-92ca-746934bbca21.png)

I think, either:
1. Fix foreground and highlight colours in non-workbench text editors
1. Don't apply colour theme to non-workbench text editors (this might be a VS Code change)

## Work around

Add the below lines to your `settings.json`. It changes the input box background colour to the same as the workbench background, so highlights are visible.

```json
    "workbench.colorCustomizations": {
        "[Monokai Dark Soda]": {
            "input.background": "#242424"
        }
    }
```

![Screenshot 2020-03-02 at 15 12 05](https://user-images.githubusercontent.com/1479528/75688909-30b25780-5c98-11ea-943f-4ece21f919dc.png)

**Comment by godrix** [comment]
there is already a PR  this fix #7

**Comment by AdamCaviness** [comment]
Merged PR , thanks.
